Makefile

```diff
@@ -98,6 +98,10 @@ releases:
 .PHONY: package
 package: clean releases
 
+.PHONY: sync
+sync: charm-helpers.yaml
```

Comment:
Unless charm-helpers.yaml might not be there, there's no need for this to be a requirement for the target.